internal/build/tar.go

```diff
@@ -31,24 +31,18 @@ func archiveDf(ctx context.Context, tw *tar.Writer, df Dockerfile) error {
 	return nil
 }
 
-// archiveIfExists creates a tar archive of all local files in `paths` _if they exist_,
-// and returns a list of all PathMappings that point to nonexistent LocalPaths (we'll
-// use that later to add RM statements).
+// archiveIfExists creates a tar archive of all local files in `paths` _if they exist_.
```

Comment:
likewise this comment is no longer really relevant.

Reply:
updated